resources/styles/global/react-datepicker.less

```diff
@@ -0,0 +1 @@
+@import '../../../node_modules/react-datepicker/dist/react-datepicker.min.css';
```

Comment:
Should we use the minified version since we are minifying for production anyway?